### ISSUE TITLE
Google login shrinks typing area

### ISSUE THREAD

**Issue by elijahr2411** [mozilla/fx-private-relay#1532]
Logging into google with relay produces a strange effect. The size of the input box stays the same but the actual typing area shrinks. See below:

![googlelogin](https://user-images.githubusercontent.com/62162399/153533850-9e22cb78-74f5-40d5-91f7-909e7d03a4dc.gif)

**Comment by tcinotto** [comment]
Thank you for filing, we'll take a look at fixing this soon!

**Comment by ckarlof** [comment]
I experience this too.

<img width="558" alt="Screenshot 2023-05-19 at 8 29 14 AM" src="https://github.com/mozilla/fx-private-relay/assets/2365255/65875230-8fd3-4d4d-85aa-812e69958b3c">

**Comment by Vinnl** [comment]
@tcinotto Yes [it](https://github.com/mozilla/fx-private-relay-add-on/pull/462) will! We just need to release it :)

**Comment by EMMLynch** [comment]
This issue is being automatically closed as it is older than 2 years and has not been addressed. If this issue is still relevant, please feel free to reopen it with updated information.
